x-pack/legacy/plugins/infra/public/components/logging/log_entry_flyout/log_entry_actions_menu.test.tsx

```diff
@@ -225,9 +221,9 @@ describe('LogEntryActionsMenu component', () => {
       expect(
         elementWrapper.find(`a${testSubject('~apmLogEntryActionsMenuItem')}`).prop('href')
       ).toMatchInlineSnapshot(
-        `"/app/apm#/traces?kuery=${encodeURIComponent(
-          'trace.id:1234567'
-        )}&rangeFrom=2019-06-27T17:34:08.693Z&rangeTo=2019-06-27T17:54:08.693Z"`
+        `"/app/apm#/link-to/trace/1234567?rangeFrom=${encodeURIComponent(
+          '2019-06-27T17:34:08.693Z'
+        )}&rangeTo=${encodeURIComponent('2019-06-27T17:54:08.693Z')}"`
```

Comment:
With the URL now being provided by the APM plugin I wonder if testing for the specific URL still makes sense. Maybe just testing for the presence of the link would be more appropriate now?

Reply:
yeah I was wondering the same. This is testing the function in itself. I will quickly tweak it and merge